valuable/Cargo.toml

```diff
@@ -1,10 +1,25 @@
 [package]
 name = "valuable"
 version = "0.1.0"
-authors = ["Carl Lerche <me@carllerche.com>"]
+authors = ["Tokio Contributors <team@tokio.rs>"]
 edition = "2018"
 license = "MIT"
-description = "Values"
+# rust-version = "1.42.0"
+readme = "README.md"
```

Comment:
If `README.md` is existing, cargo will automatically set `readme = "README.md"`: https://github.com/rust-lang/api-guidelines/pull/234

```suggestion
```